src/class-recommended-content.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Recommended_Content class file.
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
```

Comment:
How about putting this into a `Parsely\Api` namespace and corresponding file path?

Reply:
I think if we are to put it into a separate namespace, it makes sense to disambiguate the API to mean the Parse.ly service API vs the REST API on the site itself. What about `Parsely\RemoteApi` or similar...?

Reply:
:+1:  I went ahead and moved it to `Parsely\RemoteAPI` in namespace and file path.